utils/openshift_acme.py

```diff
@@ -1,76 +1,38 @@
 ACME_DEPLOYMENT = """
-apiVersion: extensions/v1beta1
 kind: Deployment
+apiVersion: apps/v1
 metadata:
   name: %(deployment_name)s
   labels:
     app: openshift-acme
 spec:
-  progressDeadlineSeconds: 600
   replicas: 1
-  revisionHistoryLimit: 10
   selector:
     matchLabels:
       app: openshift-acme
   strategy:
-    type: Recreate
+    type: RollingUpdate
   template:
     metadata:
-      creationTimestamp: null
       labels:
         app: openshift-acme
     spec:
       containers:
-      - env:
-        - name: OPENSHIFT_ACME_EXPOSER_IP
-          valueFrom:
-            fieldRef:
-              apiVersion: v1
-              fieldPath: status.podIP
-        - name: OPENSHIFT_ACME_ACMEURL
-          value: 'https://acme-v01.api.letsencrypt.org/directory'
-        - name: OPENSHIFT_ACME_LOGLEVEL
-          value: '4'
-        - name: OPENSHIFT_ACME_NAMESPACE
-          valueFrom:
-            fieldRef:
-              apiVersion: v1
-              fieldPath: metadata.namespace
-        image: %(image)s
+      - image: %(image)s
         imagePullPolicy: Always
         name: openshift-acme
-        ports:
-        - containerPort: 5000
-          protocol: TCP
         resources:
-          limits:
-            cpu: 50m
-            memory: 100Mi
-          requests:
-            cpu: 5m
+          limits:	
+            cpu: 50m	
+            memory: 100Mi	
+          requests:	
+            cpu: 5m	
             memory: 50Mi
-        terminationMessagePath: /dev/termination-log
-        terminationMessagePolicy: File
-        volumeMounts:
-        - mountPath: /dapi
-          name: podinfo
-          readOnly: true
-      dnsPolicy: ClusterFirst
-      restartPolicy: Always
-      schedulerName: default-scheduler
-      securityContext: {}
-      serviceAccount: %(serviceaccount_name)s
+        args:
+        - --exposer-image=quay.io/tnozicka/openshift-acme:exposer
```

Comment:
This version schema is so weird. There are no versions, just `quay.io/tnozicka/openshift-acme:exposer` and `quay.io/tnozicka/openshift-acme:controller`